scripts/postinstall.ts

```diff
@@ -109,14 +110,25 @@ function buildDownloadContextFor(rawPlatform: DependencyPlatform, depVersions: D
   return downloadContext;
 }
 
-runScripts().then(() => {
-  execFileSync('node', ['node_modules/electron-builder/out/cli/cli.js', 'install-app-deps'], { stdio: 'inherit' });
-  execFileSync('node', ['scripts/ts-wrapper.js',
-    'scripts/generateCliCode.ts',
-    'pkg/rancher-desktop/assets/specs/command-api.yaml',
-    'src/go/rdctl/pkg/options/generated/options.go'], { stdio: 'inherit' });
-})
-  .catch((e) => {
-    console.error(e);
-    process.exit(1);
-  });
+// The main purpose of this setTimeout is to keep the script waiting until the main async function finishes
+const keepScriptAlive = setTimeout(() => { }, 24 * 3600 * 1000);
+
+(async() => {
+  const execFileP = util.promisify(execFile);
```

Comment:
It might be easier to just import `spawnFile` from `@pkg/utils/childProcess`?

Reply:
Not exactly, but it's done.